### ISSUE TITLE
Changing item the PCM buffer is not called anymore

### ISSUE THREAD

**Issue by fabiosoft** [akhilcb/ACBAVPlayerExtension#2]
I can see the callback called goo on first item, when I change the item with method: `replaceCurrentItemWithPlayerItem `

```obj-c
    AVPlayerItem *item = [self audioItem];
    [self.player replaceCurrentItemWithPlayerItem:item];
```
the callback `audioPCMBufferFetchedWithCallbackBlock` is not called anymore.

I've also tried: 

```
    [self.player play];
    [self.player setMeteringEnabled:YES];

    [self.player audioPCMBufferFetchedWithCallbackBlock:^(AVAudioPCMBuffer *audioPCMBuffer, BOOL iSuccess) {
        
    }];
```

forcing metering enabled to YES again...but nothing changes. 

Are there come known issues with audio/video items?
How can I fix?

**Comment by akhilcb** [comment]
Apologies for the delay in replying. This should work again if you set metering enabled to YES again. I am not sure why it was happening since I was not able to reproduce this when I set it to YES again. If you found any solution, please submit a PR.

**Comment by emjoseph** [comment]
Hey, 
From the way the codebase is set up, to fix this issue do something like this to fix this issue:
```
[self.player setMeteringEnabled:NO];
[self.player replaceCurrentItemWithPlayerItem:item];
[self.player setMeteringEnabled:YES];
```
I.E make sure to turn metering off before changing the player item and then turn back on after replacement.

This behavior happens because of the **-(void)setMeteringEnabled:(BOOL)iMeterinngEnabled** method in `ACBAudioProcessHelper.m` file. 

```
- (void)setMeteringEnabled:(BOOL)iMeteringEnabled {
    
    if (!self.isMeteringEnabled && iMeteringEnabled) {
       c
    } else if (self.isMeteringEnabled && !iMeteringEnabled) {
        self.player.currentItem.audioMix = nil;
    }

    _meteringEnabled = iMeteringEnabled;
}
```
` [self setupMetering]` is never called for the new item unless you manually turn off metering and then turn it on again.

**Comment by akhilcb** [comment]
This should be fixed in v2.1. I have added a new `replaceCurrentItemAndUpdateMeteringForPlayerItem` which should do the above automatically. Please update here if it doesn't work for you.
